src/api/common.ts

```diff
@@ -1,2 +1,17 @@
+import axios from 'axios'
+
 export const BASE_URL = 'https://api.devcycle.com/'
 export const AUTH_URL = 'https://auth.devcycle.com/'
+
+export const setDVCReferrer = (
+    command: string = 'unknown',
+    version: string,
+    caller: string = 'cli'
```

Comment:
should caller only be set if the cli was called from an integration? or do we want to default to `cli`?

Reply:
I think if we default to caller being the same as the source then we can always use "caller" to understand which tools developers are interacting with — that being said, we could handle the defaulting on the API side

Reply:
I think it's fine to handle it here